antiplag/tasks.py

```diff
@@ -21,7 +21,6 @@ def process_documents(submission_id):
     submission.save()
 
     documents = submission.documents.all()
-
```

Comment:
Have you removed the empty lines on purpose or did `black` formatter do this?

Reply:
it was my bad. fixed